packages/ember-glimmer/tests/integration/helpers/inline-if-test.js

```diff
@@ -0,0 +1,150 @@
+import { moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import EmberObject from 'ember-runtime/system/object';
+import jQuery from 'ember-views/system/jquery';
+
+import {
+  BASIC_TRUTHY_TESTS,
+  BASIC_FALSY_TESTS,
+  SharedHelperConditionalsTest
+} from '../../utils/shared-conditional-tests';
+
+moduleFor('Helpers test: inline {{if}}', class extends SharedHelperConditionalsTest {
+
+  templateFor({ cond, truthy, falsy }) {
+    return `{{if ${cond} ${truthy} ${falsy}}}`;
+  }
+
+  ['@test it allows static arguments']() {
+    this.render(`{{if condition 'yes' 'no'}}`, { condition: true });
+
+    this.assertText('yes');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('yes');
+
+    this.runTask(() => set(this.context, 'condition', false));
+
+    this.assertText('no');
+  }
```

Comment:
> Isn't this the same as the BASIC_TRUTHY_TESTS? (The condition isn't really static)

I'm not sure that the `BASIC_TRUTHY_TESTS` cover this, they currently drive templates such as `{{if ${cond} ${truthy} ${falsy}}}` but not `{{if ${cond} "T" "F"}}`. This behaviour is covered by tests such as `it respects isTruthy when object changes` though, so I'll drop this test as you suggest.

> the general pattern we have been following is: .... 5. Reset (via replacement if applicable)`

I'll update the tests to include step 5 which I missed when reviewing the `with` PR